File: HASKELL/1008.hs
import Text.Printf

main::IO()
main = do
 
 number <- readLn::IO Double
 hours <- readLn::IO Double
 per_hour <- readLn::IO Double

 let salary = hours * per_hour

 printf "NUMBER = %.0f\nSALARY = U$ %.2f\n"  number salary

File: GO/1008.go
package main

import "fmt"

func Salary(hours int64, perHour float64) float64 {
	return float64(hours) * perHour
}
func main() {
	var number, hours int64
	var perHour float64

	fmt.Scan(&number)
	fmt.Scan(&hours)
	fmt.Scan(&perHour)

	fmt.Printf("NUMBER = %d\n", number)
	fmt.Printf("SALARY = U$ %.2f\n", Salary(hours, perHour))
}
